x-pack/plugins/infra/public/common/visualizations/lens/visualization_types/layers/column/formula.ts

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { FormulaPublicApi, PersistedIndexPatternLayer } from '@kbn/lens-plugin/public';
+import type { DataView } from '@kbn/data-views-plugin/public';
+import type { FormulaConfig, ChartColumn } from '../../../../types';
+
+export class FormulaColumn implements ChartColumn {
+  constructor(private formulaConfig: FormulaConfig, private formulaAPI: FormulaPublicApi) {}
+
+  getName() {
+    return this.formulaConfig.label ?? '';
```

Comment:
Q: Do we fallback to the formula value if we don't have a Label ( I think it's like that in Lens or maybe I am mistaken) 
<img width="429" alt="image" src="https://github.com/elastic/kibana/assets/14139027/a88a5208-2a94-4dc5-8264-c1d93e4fef17">

Reply:
I've changed it to return `undefined` in case a custom label is not defined. When `label`, Lens will use the formula as the fallback